terms_of_use/templates/terms_of_use/components/base.jinja

```diff
@@ -0,0 +1,229 @@
+  tMS45ODRsMS44NzMtMS4wNTVjLjIzNi44MzUuOTEzIDEuNDY0IDEuOTM2IDEuNDY0LjU4My4wMTYgMS4wMDgtLjIzNiAxLjAwOC0uNzI0IDAtLjYxNC
+  0uODUtLjc0LTEuODQyLTEuMDIzLTEuMTY1LS4zLTIuNTk3LS44ODItMi41OTctMi4zNDUgMC0xLjQ0OSAxLjM4NS0yLjM2MiAzLjAzOC0yLjM2MnpNM
+  zIuMyAxMC41M3Y4LjUwMWMwIC4zNjIuMDQ3LjYxNC4xNTcuNzU2LjA5NS4xNTcuMjIuMjIuNTA0LjIyLjE3MyAwIC4zMyAwIC40NTYtLjAzMWEyLjAx
+  IDIuMDEgMCAwMC40NTctLjEyNmwtLjE4OSAxLjQ0OGEyLjUzMiAyLjUzMiAwIDAxLS42NDUuMjM2Yy0uMjUyLjA0Ny0uNTIuMDc5LS43NzIuMDc5LS4
+  3NCAwLTEuMjc1LS4xNzQtMS42MjEtLjUzNi0uMzMtLjM2Mi0uNTA0LS45NDQtLjUwNC0xLjczMVYxMC41M2gyLjE1N3ptMTkuODE4IDMuMDg2YzEuND
+  ggMCAyLjQwOCAxLjAyMyAyLjQwOCAyLjkyOHY0LjkxMWgtMi4xNTZWMTYuOTdjMC0xLjAwNy0uMzYyLTEuNTQzLTEuMTY1LTEuNTQzcy0xLjMyMi42M
+  TQtMS4zMjIgMS40OTZ2NC41NDloLTIuMTQxdi03LjY2NmgyLjE0bC0uMTEgMS4yNmguMDQ4Yy4zNzgtLjcxIDEuMDM5LTEuNDQ5IDIuMjk4LTEuNDQ5
+  em0xNi43MzMtMy4zMzdhMy4yNyAzLjI3IDAgMDEyLjczOSAxLjRsLTEuODU4IDEuMTM0Yy0uMDMxLS40NTYtLjI1Mi0uOTI4LS43NzEtLjkyOC0uNTM
+  1IDAtLjc4Ny40MDktLjc4NyAxLjA4NnYuODM0aDEuNjg0djEuNjM3aC0xLjY4NHY2LjAyOWgtMi4xNHYtNi4wMjloLTEuMTgxdi0xLjYzN2gxLjE4di
+  0uODE5YzAtMS43OTQgMS4xOC0yLjcwNyAyLjgxOC0yLjcwN3ptMTQuODEyIDMuMzg0YzEuMDg2IDAgMS42ODQuOTEzIDEuNjg0IDIuMDMgMCAuNTA1L
+  S4wNzguODk4LS4wNzguODk4bC0xLjkzNiAxLjExOGMuMDMxLS4yMi4xMjYtLjc0LjEyNi0xLjI2IDAtLjUwMy0uMTktLjk3NS0uNzI1LS45NzUtLjY3
+  NiAwLS45NDQuNjMtLjk0NCAxLjYydjQuMzc3aC0yLjE1N3YtNy42NWgyLjE1N2wtLjExIDEuMjU5aC4wNDdjLjM3OC0uNzQuOTYtMS40MTcgMS45MzY
+  tMS40MTd6bS03MC41Mi0yLjcyM3Y0LjE3MWg0LjIwM3YtNC4xN2gyLjIzNXYxMC41MTRoLTIuMjM1di00LjM2aC00LjIwM3Y0LjM2aC0yLjIzNVYxMC
+  45NGgyLjIzNXptMzIuMzMyIDIuODV2Ny42NjVINDMuMzJWMTMuNzloMi4xNTZ6bTI5LjE1MyAxLjQ2M2MtMS4wNyAwLTEuNTQzIDEuMDA4LTEuNTQzI
+  DIuMzc3IDAgMS4zNy40NzIgMi4zNzcgMS41NDMgMi4zNzcgMS4wNTQgMCAxLjUyNy0xLjAwNyAxLjUyNy0yLjM3NyAwLTEuMzctLjQ3My0yLjM3Ny0x
+  LjUyNy0yLjM3N3ptLTE0Ljg5MS0uMTg5Yy0uNzI0IDAtMS4zMDcuNDczLTEuMzA3IDEuMTk3IDAgLjc0LjU4MyAxLjIxMiAxLjMwNyAxLjE5Ni43MjQ
+  gMCAxLjMwNi0uNDcyIDEuMzA2LTEuMTk2cy0uNTgyLTEuMTk3LTEuMzA2LTEuMTk3em0tMzQuNzczLjA5NWMtLjc4NyAwLTEuNDMyLjU2Ni0xLjYwNS
+  AxLjYwNWgzLjAzOGMwLS45MTMtLjU5OC0xLjYwNS0xLjQzMy0xLjYwNXptMTkuNDQtNC42NzVjLjY5MyAwIDEuMjQ0LjUwMyAxLjI0NCAxLjE5NnMtL
+  jU1IDEuMTk2LTEuMjQzIDEuMTk2Yy0uNzA5IDAtMS4yNDQtLjUwMy0xLjI0NC0xLjE5NnMuNTUxLTEuMTk2IDEuMjQ0LTEuMTk2eiIKICAgICAgICBm
+  aWxsPSJjdXJyZW50Q29sb3IiIC8+Cjwvc3ZnPg==
+  " %}
+
+  {% set logo_en = logo_fi %}
+
+  {% for lang in ['fi', 'sv', 'en'] %}
+
+    {% if lang == 'fi' %}
+      {% set logo = logo_fi %}
+      {% set city_heading = 'HELSINGIN KAUPUNKI' %}
+      {% set government_heading = 'Keskushallinto' %}
+
+    {% elif lang == 'sv' %}
+      {% set logo = logo_sv %}
+      {% set city_heading = 'HELSINGFORS STAD' %}
+      {% set government_heading = 'Centralförvaltningen' %}
+
+    {% else %}
+      {% set logo = logo_en %}
+      {% set city_heading = 'CITY OF HELSINKI' %}
+      {% set government_heading = 'Central government' %}
```

Comment:
Looks like the term which is used in EN translation is 'Central Administration'
Cf. e.g. https://www.hel.fi/en/decision-making/city-organisation/divisions